django_event_sourcing/registers.py

```diff
@@ -37,15 +38,37 @@ def decorator(f):
 
         return decorator
 
-    def handle(self, event):
+    def register_side_effect(self, *, side_effect):
```

Comment:
```suggestion
    def register_side_effect(self, side_effect):
```

I don't think this needs to be a keyword because the function only takes one argument.

Reply:
Im gonna put conditions in soon so might keep it